src/rendering/src/components/Search/SearchResultsContainer.tsx

```diff
@@ -0,0 +1,104 @@
+import { useQuery } from '@tanstack/react-query';
+import React, { useMemo } from 'react';
+import { ContentSearchResponseBase } from '../../interfaces/contentSearch/ContentSearchResponse';
+import connectResultsTab from '../../lib/contentSearch/hocs/connectResultsTab';
+import * as api from '../../lib/contentSearch/api';
+import SearchNewsResultsTab from './SearchNewsResultsTab';
+import SearchSessionResultsTab from './SearchSessionResultsTab';
+import SearchSpeakerResultsTab from './SearchSpeakerResultsTab';
+import SearchSponsorResultsTab from './SearchSponsorResultsTab';
+import SearchVendorResultsTab from './SearchVendorResultsTab';
+import { SearchFiltersProps } from './SearchFilters';
+import SearchProvider from './SearchProvider';
+import SearchResults from './SearchResults';
+
+type SearchResultsContainerProps = {
+  q: string;
+  tab: string;
+};
+
+const widgetId = 'rfkid_7';
+
+const tabs = [
```

Comment:
Hmm. Do you mean to show show results of sessions, speakers, vendors, sponsors and news in the same tab? Different card types in the same results tab would look weird, wouldn't they?

Reply:
Yes that is what I mean. I would like all sessions, speakers, vendors, sponsors, news, and other site pages in that tab. Maybe there could be a different harmonized card type for that specific tab. Like rows with optional image on the left, title on the right plus some other details if available.

Reply:
What if we automatically select the first tab with results > 0?

Reply:
I've just talked with who is in charge of the API and unfortunately this is not possible, it's not supported and seems like a use case like this one won't be supported in the future either.

Reply:
What exactly is not possible? The "All results" tab, or the different result card template per tab?

Reply:
My bad. Here's why: We're currently calling the API once per tab (filtering by entity) for the given query. To implement the "All results" tab were going to make an extra request without the entity filtering to get results for all the entities, allowing us to have facets, pagination and sorting. The API doesn't have that support.

I wouldn't compile the list of items based on the entity tab results because we lose relevance (eg. which ones should be shown first?), sorting (sorting options vary by entity), facets and pagination.

Reply:
- Which ones should be shown first: A healthy mix of the search results with the most searched keywords occurrences, the title most matching the searched keywords, the most recent updated date. This is basic search engine relevancy sorting. Then as Sitecore Search is an intelligent search solution, you can also mix the search results with the most clicks by other users for similar queries, the most potential to be relevant for me based on ML, etc.
- Facets: Show all facets. If a facet applies to only one result type, this is not a problem. Using it would only show results of that type and it is fine.
- Sorting: On that all results tab, there can be very limited sorting options: Relevance, A-Z, Z-A.
- Pagination: This is the same as with any other tab as far as I know.

Sitecore Search should really support creating a search page that shows all results. This is a basic feature of any search engine and the mode that gives the best relevancy to the website visitor. If this is really not possible, please tell me which product manager to talk to for discussing this topic. Thanks

Reply:
I agree 100%.

Since that is not supported by the API, I was suggesting to not do that from frontend side.

Reply:
Alright. Let's not do it for now.